src/components/Dynamic.vue

```diff
@@ -44,6 +45,10 @@ export default {
       type: String,
       default: null,
     },
+    componentRef: {
+      type: String|Number,
```

Comment:
since the module is still vue `v2` the ref needs to be a `string`: https://vuejs.org/v2/api/#ref

Reply:
Maybe it is casted internally because it works with numbers as well. I am not sure... 

This is my use case and it works perfectly:

```vue
<template>
  <div>
    <div v-for="(tab, i) in tabs" :key="tab.id">
      <div v-show="i == currentTab">
        <NuxtDynamic :component="tab.componentName" :componentRef="i" :ref="i" :tab="tab" />
      </div>
    </div>
  </div>
</template>

<script>
export default {
  computed: {
    currentTab() {
      this.$refs[this.$store.state.tabs.currentTab]?.[0]?.$refs[this.$store.state.tabs.currentTab]?.activated()
      return this.$store.state.tabs.currentTab
    },
    tabs() {
      return this.$store.state.tabs.tabs
    },
  },
}
</script>
```
As you can see i use the index of the for loop for both refs...

Reply:
alright - let's leave it as it is then 👍🏼